lib/restrictedsession/restricted_test.go

```diff
@@ -220,7 +220,7 @@ func setupBPFContext(t *testing.T) *bpfContext {
 	bpfCtx.restrictedMgr, err = New(config, client)
 	require.NoError(t, err)
 
-	client.Fanout.SetInit()
+	client.Fanout.SetInit(nil)
```

Comment:
Shouldn't this have a `confirmedKinds`? Is an empty list of confirmed kinds allowed?

Reply:
Right. I can't run this test locally because I'm on a mac, but I'm surprised it didn't fail on CI. Presumably fixed in https://github.com/gravitational/teleport/pull/24080/commits/22d4efd5dc0ace03172d6d74eb717d71932490a3